docs/en-US/component/drawer.md

```diff
@@ -116,7 +116,7 @@ Drawer provides an API called `destroyOnClose`, which is a flag variable that in
 | ----------------- | ---------------------------------------------------------------------------------------------- |
 | —                 | Drawer's Content                                                                               |
 | header            | Drawer header section; Replacing this removes the title, but does not remove the close button. |
-| title(deprecated) | Works the same as the header slot. Use that instead.                                           |
+| title <DeprecatedTag /> | Works the same as the header slot. Use that instead.                                           |
```

Comment:
Thanks for your contribution, `DeprecatedTag` has its own spacing, so you can remove the preceding space, and then do you mind executing `pnpm run lint:fix` again?

Reply:
done